dashboard/app/models/lesson.rb

```diff
@@ -13,6 +13,7 @@
 #  properties        :text(65535)
 #  lesson_group_id   :integer
 #  key               :string(255)      not null
+#  has_lesson_plan   :boolean
```

Comment:
Out of curiousity, why does this need to be in the table instead of in `properties`?

Reply:
I think because we will end up using it to determine things like the URL of the page which means it needs to be in the table.

Reply:
That seems like a good reason. Just a heads up, to make seeding work for this new field, you'll need to also add it here: https://github.com/code-dot-org/code-dot-org/blob/84718a78f9d506ee00e05e19fc8bb2cc698161e0/dashboard/lib/services/script_seed.rb#L442-L441

Reply:
Makes sense!